packages/parser/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "./dist",
+    // TODO: figure out how to remove these configs
+    "preserveSymlinks": false,
+    "strictPropertyInitialization": false
+  },
+  "include": ["./package.json", "./src/**/*.ts", "./tests/**/*.ts"],
```

Comment:
@sidharthv96 How to handle `"./tests/**/*.ts"`?
Removing it would make TS server to throw errors since it can't import `langium`.
And leaving it here would make `tsc` compile and add them to `dist`.

Reply:
Why is tsc used to compile? Isn't esbuild the recommended tool?

Reply:
I was using at the beginning just to get the parser running. If you have time, feel free to configure it since I'm sure how to :)

Reply:
@sidharthv96 I've used `esbuild`, please review it and check if it's correct.

But we are missing `package.json` in `packages/parser/dist`, and still `tests` types are generated.